packages/web3/types/index.d.ts

```diff
@@ -0,0 +1,53 @@
+/*
+    This file is part of web3.js.
+    web3.js is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    web3.js is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+    You should have received a copy of the GNU Lesser General Public License
+    along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/**
+ * @file index.d.ts
+ * @author Josh Stevens <joshstevens19@hotmail.co.uk>
+ * @date 2018
+ */
+
+import {AbstractWeb3Module, Web3ModuleOptions, Providers} from 'web3-core';
+import {Utils} from 'web3-utils';
+import * as net from 'net';
+import {provider, AbstractProviderAdapter, BatchRequest} from 'web3-providers';
+import {Eth} from "web3-eth";
+import {Network} from "web3-net";
```

Comment:
Could you run eslint for the typing files to remove the double quotes to single quotes and other stuff? :)

Reply:
oops normally do will sort now